fhttp/http.go

```diff
@@ -53,8 +53,10 @@ type Fetcher interface {
 }
 
 var (
-	// BufferSizeKb size of the buffer (max data) for optimized client in kilobytes defaults to 32k.
+	// BufferSizeKb size of the buffer (max data) for optimized client in kilobytes defaults to 128k.
 	BufferSizeKb = 128
+	// MaxPayloadSize is the maximum size of payload to be generated by echo handler defaults to 256k
```

Comment:
missing " in bytes" and "size= argument" from the original comment
In theory as there is a ChangeXxxx function to change it, the variable should be private
eg `maxPayloadSize` but we do need the default value for the flag... well or do we the flag default can be whichever

Reply:
Using one default in flag and another in http_server file doesn't sound good ? I have kept  MaxPayloadSize public for now